language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/CommandUtil.java

```diff
@@ -467,4 +540,169 @@ int getReplaceStartFrom() {
             return replaceStartFrom;
         }
     }
+
+    /**
+     * Inner class for generating function code.
+     */
+    public static class FunctionGenerator {
+
+        /**
+         * Generate function code.
+         *
+         * @param name               function name
+         * @param returnType         return type
+         * @param returnDefaultValue default return value
+         * @return
+         */
+        public static String createFunction(String name, String returnType, String returnDefaultValue) {
+            String funcBody = "\r\n";
```

Comment:
Let's use System.lineSeparator() instead of \r\n. You can refer via CommonUtil.LINE_SEPARATOR